features/README.md

```diff
@@ -2,7 +2,7 @@ This gem provides missing testing utils for [Action Cable][].
 
 **NOTE:** this gem is just a combination of two PRs to Rails itself ([#23211](https://github.com/rails/rails/pull/23211) and [#27191](https://github.com/rails/rails/pull/27191)) and (hopefully) will be merged into Rails eventually.
 
-**NOTE 2:** this is the documentation for (mostly) RSpec part of the gem. For Minitest usage see the repo's [Readme](https://github.com/palkan/action-cable-testing) or [Minitest Usage](minitest) chapter.
+**NOTE 2:** this is the documentation for (mostly) RSpec part of the gem. For Minitest usage see the repo's [Readme](https://github.com/palkan/action-cable-testing) or [Minitest Usage](Minitest.md) chapter.
```

Comment:
The link was correct; see https://relishapp.com/palkan/action-cable-testing/docs/minitest

Reply:
@palkan Oh I see, thanks. I javen't thought about relishapp